templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Same here, the output can be cleared.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Are you sure? This is the same output in the current version of the mesmer-only quantification notebook</p>

Reply:
<p>Doesn't look like the output is there anymore in the second-to-last cell (<a href="https://github.com/angelolab/ark-analysis/blob/main/templates/1_Segment_Image_Data.ipynb" target="_blank">https://github.com/angelolab/ark-analysis/blob/main/templates/1_Segment_Image_Data.ipynb</a>), so I thought it best to remove.</p>